### PR TITLE
Some minor modifications

### PR DESCRIPTION
**Pull request by mohamadyounes33**
- Loggers are displayed from each worker
- Deleted useless MPI communication that contained valid_loss, accuracy,... to the master.

### CODE REVIEW

File: diagnosenet/executors.py [hunk 1]
```diff
@@ -1442,7 +1442,8 @@ def set_dataset_disk(self, dataset_name: str, dataset_path: str,
                 train, valid, test = self.data.disk_one_target()
             elif 'Batching' in str(type(self.data)):
                 if self.rank != 0:
-                    train, valid, test = self.data.distributed_batching(dataset_name, self.job_name, self.task_index)
+                    train, valid, test = self.data.distributed_batching(dataset_name, self.job_name,
+                                                                        self.task_index - 1)
                 else:
                     train, valid, test = self.data.distributed_batching(dataset_name, self.job_name,
```
Comment (jagh) [inline diagnosenet/executors.py:1452]: To ensure MPI batching integrity in future improvements, we keep the same rank for the master `task_index`. 
```
if self.rank != 0:
     train, valid, test = self.data.distributed_batching(dataset_name, self.job_name,  self.task_index - 1)
else:
     train, valid, test = self.data.distributed_batching(dataset_name, self.job_name,  self.task_index)
```

File: diagnosenet/executors.py [hunk 2]
```diff
@@ -1788,10 +1772,11 @@ def synchronous_training(self, dataset_name: str, dataset_path: str,
                     logger.info(
```
Comment (jagh) [inline diagnosenet/executors.py]: In `Astro` array server, the logger has some errors of indentation when the line is divided, also for the `self.training_track.append` line.
```
logger.info("Epoch {} | Train loss: {} |  Valid loss: {} | Train Acc: {} | Valid Acc: {} | Epoch_Time: {}".format(epoch, loss, valid_loss, acc, val_acc, np.round(epoch_elapsed, decimals=4)))
```

File: diagnosenet/executors.py [hunk 3]
```diff
@@ -1578,17 +1576,12 @@ def asynchronous_training(self, dataset_name: str, dataset_path: str,
                                              y_pred=valid_pred.astype(np.float), average='micro')
                         val_acc += valid_acc / len(valid.input_files)
                         valid_loss += valid_loss / len(valid.input_files)
-
-                epoch_elapsed = (time.time() - epoch_start)
-                if self.rank == 0:
-                    val_acc, valid_loss = self.comm.recv(source=MPI.ANY_SOURCE, status=self.status, tag=2)
+                    epoch_elapsed = (time.time() - epoch_start)
                     logger.info(
```
Comment (jagh) [inline diagnosenet/executors.py]: In `Astro` array server, the logger has some errors of indentation when the line is divided, also for the `self.training_track.append` line.
```
 logger.info("From {} | Epoch {} | Train loss: {} |  Valid loss: {} | Train Acc: {} | Valid Acc: {} | Epoch_Time {}".format(self.rank, epoch, train_loss, valid_loss, acc, val_acc, np.round(epoch_elapsed, decimals=4)))
```

**Review by jagh — CHANGES_REQUESTED** [review]

**Review by jagh — APPROVED** [review]

**Comment by jagh** [comment]
Updated the MPI asynchronous logger displayed by each worker training
